package.json

```diff
@@ -89,6 +89,6 @@
         "not op_mini all",
         "not < 1%",
         "Chrome 77",
-        "iOS 12"
+        "iOS 13.4"
```

Comment:
That's the minimum version to support ResizeObserver. If the editor makes it AnkiMobile before 13.4 is more spread, we could just generally turn off the image resizing and Mathjax editor for AnkiMobile (that's the two cases were we use `ResizeObserver`).

Reply:
While it will probably be a while until I get the editor ported over, a bump to 13.4 is not too far off, and may come in the next update, so this should be fine. :+1: